.env.example

```diff
@@ -76,6 +77,12 @@ STORAGES='{
 # DEFAULT: ""
 # STORAGES_PROJECT_DEFAULT_STORAGE=
 
+# Local admin credentials configuration for minio storage and standalone instances. Uncomment if necessary.
+# NOTE: Needs to be the same as in the `STORAGES` setting in standalone config.
+# DEFAULT: MINIO_ROOT_USER=minioadmin, MINIO_ROOT_PASSWORD=minioadmin
+# MINIO_ROOT_USER=minioadmin
+# MINIO_ROOT_PASSWORD=minioadmin
```

Comment:
Would you mind ungrouping the envvars, as we do with any other? I can see the logic why you propose them merged, but I would like to keep one style. And slowly add this documentation for all the other envvars :)

Reply:
Done